data/json/items/chemicals_and_resources.json

```diff
@@ -890,5 +890,24 @@
     "weight": 212,
     "volume": 1,
     "to_hit": -2
-    }
+    },{
+    "type" : "COMESTIBLE",
```

Comment:
> No one in right senses won't drink salt water or eat fertilizer, but they're still comestibles.

That's the wrong approach. Salt water and fertilizer should lose their comestible status, not be used as excuses for other things to be noob traps polluting the 'E'at menu. They are comestibles because some liquid code depended (still depends?) on things being comestibles, not because someone had a horrible idea of making them comestibles just like that (at least I hope no one has ideas this horrible).

Though sewage water should actually be a comestible since otherwise it will be totally useless. It has a `>0` quench and no use other than drinking.